modules/lookup.js

```diff
@@ -92,27 +97,81 @@ function lookupTerm(message) {
-            .then(msg => {
-                tmp.setGracefulCleanup();
-            })
-            .catch(error => {
-                console.error(error);
-                tmp.setGracefulCleanup();
-            });
-        })
-        .catch(console.error);
+    
+    // temp: test no-sandbox args
+    // args: ['--no-sandbox', '--disable-setuid-sandbox'] - does not work
+    
+    // launchOptions: {args: ['--no-sandbox', '--disable-setuid-sandbox']} - this is what needs to be passed to pageres -> capture-website, 
+    // but pageres filters it out in finalOptions
+    // https://github.com/sindresorhus/pageres/blob/master/source/index.ts
+    // https://www.npmjs.com/package/capture-website
+    
+    let url = 'https://pf2.easytool.es/index.php?id='+ID,
+        dest = tmpdir.name,
+        filename = 'foo.png'
+        finalOptions = {
+            delay:0,
+            width:1024,
+            height:768,
+//             timeout: options.timeout, //todo
+            fullPage: false, //!options.crop
+//             styles: options.css && [options.css],
+//             scripts: options.script && [options.script],
+//             cookies: options.cookies, // TODO: Support string cookies in capture-website
+            element: 'article.result',
+//             hideElements: options.hide,
+            scaleFactor: 1,
+            type: 'png',
+//             userAgent: options.userAgent,
+//             headers: options.headers,
+            launchOptions: {
+                args: [
+                    '--no-sandbox',
+                    '--disable-setuid-sandbox'
+                ]
+            }
+        };
+    
+    const screenshot = await captureWebsite.buffer(url, finalOptions);
+    
+    await makeDir(dest);
+    
+    const fullPath = path.join(dest, filename);
+    
+    await writeFile(fullPath, screenshot);
+    
+    console.log('Saving to ' + dest);
+    message.channel.send({files: [{attachment: tmpdir.name+'/' + filename,name:'results.png'}]})
+    .then(msg => {
+        tmp.setGracefulCleanup();
+    })
+    .catch(error => {
+        console.error(error);
+        tmp.setGracefulCleanup();
+    });
+    
+    
+//     let pageres = new Pageres({delay: 0, selector:'article.result', filename:'foo'})
+//         .src('https://pf2.easytool.es/index.php?id='+ID, ['1024x768'], {crop: true})
+//         .dest(tmpdir.name)
+//         .run()
+//         .then(function() {
+//             console.log('Saving to ' + tmpdir.name+'/'+'foo.png');
+//             message.channel.send({files: [{attachment: tmpdir.name+'/'+'foo.png',name:'results.png'}]})
+//             .then(msg => {
+//                 tmp.setGracefulCleanup();
+//             })
+//             .catch(error => {
+//                 console.error(error);
+//                 tmp.setGracefulCleanup();
+//             });
+//         })
+//         .catch(console.error);
```

Comment:
Likewise this can be removed. Git is great at tracking history. If we ever need to go back to the old way, it's a simple git command.

Reply:
Removed all the comments and pageres code